code/modules/antagonists/spy/spy.dm

```diff
@@ -130,30 +130,7 @@
 		your_mission.owner = owner
 		your_mission.explanation_text = pick_list_replacements(SPY_OBJECTIVE_FILE, "objective_body")
 		objectives += your_mission
-	
-	if((length(objectives) < 3) && prob(25))
-		switch(rand(1, 4))
-			if(1)
-				var/datum/objective/protect/save_the_person = new()
-				save_the_person.owner = owner
-				save_the_person.no_failure = TRUE
-				objectives += save_the_person
```

Comment:
Mentioned in discord but I'm pretty sure they were showing nothing because you never gave them any targets

Reply:
I'll try that tonight, that would be a problem - that's odd because it was autopopulating after roundstart in testing, let me check the args

Reply:
amazingly that did it